### PR TITLE
Auto join hack

### PR DESCRIPTION
**Pull request by josephgarner**
Updated AddHack

- Added team array to hack allowing the user to be added to the hack upon creation

### CODE REVIEW

File: addHack.js [hunk 1]
```diff
@@ -19,23 +19,24 @@ export const add = async (event, context) => {
...
   }
   const Hack = conn.model("Hack");
 
   try {
-    const newHack = new Hack({ title, description, goal });
-    await newHack.save();
-
+    const newHack = new Hack({ title, description, goal, team });
+    const query = await newHack.save();
+    console.log(query._id);
```
Comment (jpeyper) [inline addHack.js]: not sure you want this anymore

File: getHacks.js [hunk 2]
```diff
@@ -18,16 +18,29 @@ export const list = async (event, context) => {
...
   const Query = conn.model("Hack");
+  const Teams = conn.model("User");
+
   /**
    * To be used in the future This will return One document
    * const doc = await Query.findOne({_id: '5e6094446a56971ad6a32d7b'});
    */
   try {
     const doc = await Query.find();
+    const users = await Teams.find();
```
Comment (jpeyper) [inline getHacks.js]: is this from an older version of the other PR?

Comment (josephgarner) [reply getHacks.js]: Older version Yes

File: addHack.js (hunk 1 above)
Comment (josephgarner) [reply addHack.js]: Very true, this has been removed

File: getHacks.js (hunk 2 above)
Comment (jpeyper) [reply getHacks.js]: can you rebase the commits for this PR off master?

**Review by jpeyper — APPROVED** [review]

**Pull request merged**